src/net/http.rs

```diff
@@ -304,59 +188,28 @@ pub(crate) async fn http_handler(
     .await
 }
 
-#[test]
-fn test_rpc_payload_serialzation_and_deserialization() {
-    let json_payload = json!({
-        "method": "dummy-value",
-        "params": [],
-        "id": 1,
-        "jsonrpc": "2.0",
-        "signed_message": {
-            "coin_ticker": "ETH",
-            "address": "dummy-value",
-            "timestamp_message": 1655319963,
-            "signature": "dummy-value",
-         }
-    });
-
-    let actual_payload: RpcPayload = serde_json::from_str(&json_payload.to_string()).unwrap();
-
-    let expected_payload = RpcPayload {
-        method: String::from("dummy-value"),
-        params: json!([]),
-        id: 1,
-        jsonrpc: String::from("2.0"),
-        signed_message: SignedMessage {
-            coin_ticker: String::from("ETH"),
-            address: String::from("dummy-value"),
-            timestamp_message: 1655319963,
-            signature: String::from("dummy-value"),
-        },
-    };
-
-    assert_eq!(actual_payload, expected_payload);
-
-    // Backwards
-    let json = serde_json::to_value(expected_payload).unwrap();
-    assert_eq!(json_payload, json);
-    assert_eq!(json_payload.to_string(), json.to_string());
-}
-
 #[test]
 fn test_get_proxy_route_by_inbound() {
+    use hyper::Uri;
+    use std::str::FromStr;
+
     let cfg = ctx::get_app_config_test_instance();
 
-    let proxy_route = cfg
-        .get_proxy_route_by_inbound(String::from("/test"))
-        .unwrap();
+    // If we leave this code line `let proxy_route = match cfg.get_proxy_route_by_inbound(req.uri().to_string()) {`
+    // inbound_route cant be "/test", as it's not uri. I suppose inbound actually should be a Path.
+    // Two options: in `req.uri().to_string()` path() is missing or "/test" in test is wrong and the whole url should be.
+    let proxy_route = cfg.get_proxy_route_by_inbound("/test").unwrap();
```

Comment:
Is this comment still needed? I see that you already use `path()`

Reply:
oh, yes, should remove it, as it was supposed to be an important note for reviewers on pr's early stages.
fixed it https://github.com/KomodoPlatform/komodo-defi-proxy/pull/22/commits/226db21f41c0461ecdb45499daca1467d7fc199c